amr-wind/equation_systems/icns/icns_ops.cpp

```diff
@@ -78,15 +80,28 @@ void advection_mac_project(FieldRepo& repo, const FieldState fstate, const bool
     amrex::LPInfo lp_info;
     lp_info.setMaxCoarseningLevel(options.max_coarsen_level);
 
+    auto div = repo.create_scratch_field(1, 0);
+    if(has_overset){
+        // fixme what should we do about density?
```

Comment:
The `div( grad(p) )` term should have the same coefficients as the `\beta` here: https://amrex-codes.github.io/amrex/docs_html/LinearSolvers.html#mac-projection